src/main/kotlin/com/sourcegraph/cody/agent/protocol/ChatModelParams.kt

```diff
@@ -0,0 +1,3 @@
+package com.sourcegraph.cody.agent.protocol
+
+data class ChatModelsParams(val id: String)
```

Comment:
Just noticed: The file (ChatModelsParams.kt) has a different name than the class name.

ChatModelsResponse.kt too.